src/solana/rpc/api.py

```diff
@@ -16,7 +16,7 @@
 from .core import (
     RPCException,
     TransactionExpiredBlockheightExceededError,
-    TransactionUncompiled,
+    TransactionUncompiledError,
```

Comment:
@crypt0miester this change here is causing the tests to fail. Why is the rename here necessary?

Reply:
wanted to use the same name convention wherein something raised has Error as suffix. I'll double check.

Reply:
weirdly enough, the make lint did not catch it. I fixed this with the latest commit.